src/animated.tpp

```diff
@@ -78,12 +78,8 @@ inline void animated<T>::start_animation(int start_time, bool cycles)
 	started_ = true;
 	last_update_tick_ = get_current_animation_tick();
 	acceleration_ = 1.0; // assume acceleration is 1, this will be fixed at first update_last_draw_time
-	start_tick_ = last_update_tick_ + static_cast<int>((starting_frame_time_ - start_time) / acceleration_);
-
+	start_tick_ = last_update_tick_ + (starting_frame_time_ - start_time);
 	cycles_ = cycles;
-	if(acceleration_ <= 0) {
```

Comment:
It is pretty late here, but I'm not missing something here, right?

animated is set to 1, at line 80, then never changed and then checked for <= 0

I guess it was a simple merge request gone weird

Reply:
Your change looks good to me. Looks as if this should have been removed in 20bf7fa2e9b56df39ee53667e96200f9d4f51f82, when this function stopped taking the acceleration as a parameter.